tsconfig.json

```diff
@@ -6,7 +6,7 @@
 
     // Modules
     "baseUrl": "./app/javascript/src",
-    "module": "ES6",
+    "module": "ES2020",
```

Comment:
I'm not much into this. I just noticed some online recommendation for [node 14 settings](https://stackoverflow.com/a/61305579/520567):
```
{
  "compilerOptions": {
    "allowSyntheticDefaultImports": true,
    "lib": ["ES2020"],
    "module": "ES2020",
    "moduleResolution": "node",
    "target": "ES2020"
  }
}
```

But then I read docs https://www.typescriptlang.org/docs/handbook/esm-node.html

So I guess that might be backward incompatible to change module type. Just throwing it out here. I guess you have considered these already.